### PR TITLE
Fix Singleton Interest

### PR DESCRIPTION
**Pull request by yumnuska**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Add Singleton and SingletonManager to QBI system constraints. 

When an entity uses QBI, it will be excluded from chunk-based interest systems (such as streaming queries). 

#### Release note
Added to changelog.

#### Tests
Validated behavior in dev and test config, verified Interest component in inspector.

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp [hunk 1]
```diff
@@ -209,6 +215,25 @@ QueryConstraint InterestFactory::CreateAlwaysInterestedConstraint()
...
 }
 
+
+QueryConstraint InterestFactory::CreateSingletonConstraint()
+{
+	QueryConstraint SingletonConstraint;
+
+	Worker_ComponentId SingletonComponentIds[] = {
+		SpatialConstants::SINGLETON_COMPONENT_ID,
+		SpatialConstants::SINGLETON_MANAGER_COMPONENT_ID };
```
Comment (m-samiec) [inline SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp]: nit: The singleton manager is only required on servers, the client early-outs all it's logic. Might be worth switching on if this is being attached to the RPC client end point, or the position component.

Comment (yumnuska) [reply SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp]: I was curious about whether it was needed or not, but it's in the streaming queries for the client which is why I included it. Do you think it's worth making that change now, or as part of the larger refactor to apply interest to a single entity per worker?

Comment (m-samiec) [reply SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp]: Yep, can refactor when working in the larger interest updates.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased-`0.5.0`] - 2019-xx-xx
 - Prevented `Spatial GDK Content` from appearing under Content Browser in the editor, as the GDK plugin does not contain any game content.
+- Added Singleton and SingletonManager to QBI constraints to fix issue preventing Test configuration builds from functioning correctly.
```
Comment (m-samiec) [inline CHANGELOG.md]: This should go in the bugfix section.

Comment (yumnuska) [reply CHANGELOG.md]: done.

**Pull request merged**